x-pack/plugins/security_solution/public/alerts/components/alerts_table/alerts_utility_bar/index.tsx

```diff
@@ -42,25 +45,69 @@ const AlertsUtilityBarComponent: React.FC<AlertsUtilityBarProps> = ({
   clearSelection,
   totalCount,
   selectedEventIds,
-  isFilteredToOpen,
+  currentFilter,
   selectAll,
   showClearSelection,
   updateAlertsStatus,
 }) => {
   const [defaultNumberFormat] = useUiSetting$<string>(DEFAULT_NUMBER_FORMAT);
 
-  const handleUpdateStatus = useCallback(async () => {
-    await updateAlertsStatus({
-      alertIds: Object.keys(selectedEventIds),
-      status: isFilteredToOpen ? FILTER_CLOSED : FILTER_OPEN,
-    });
-  }, [selectedEventIds, updateAlertsStatus, isFilteredToOpen]);
+  const handleUpdateStatus = useCallback(
+    async (selectedStatus: Status) => {
+      await updateAlertsStatus({
+        alertIds: Object.keys(selectedEventIds),
+        status: currentFilter,
+        selectedStatus,
+      });
+    },
+    [currentFilter, selectedEventIds, updateAlertsStatus]
+  );
 
   const formattedTotalCount = numeral(totalCount).format(defaultNumberFormat);
   const formattedSelectedEventsCount = numeral(Object.keys(selectedEventIds).length).format(
     defaultNumberFormat
   );
 
+  const UtilityBarPopoverContent = (closePopover: () => void) => (
+    <EuiFlexGroup style={{ minWidth: 175 }} direction="column">
```

Comment:
Isn't the typical pattern to extend a EuiFlexGroup, add the CSS and then use that component?

Lots of examples when I search for: `styled(EuiFlexGroup)`

Like this one:

```ts
export const PropertiesLeftStyle = styled(EuiFlexGroup)`
  width: 100%;
`;
```

I don't see many inline styles right now. The ones I do see have good comments about why they can't use styled to override like this one:

```ts
    <EuiHorizontalRule
      style={{
        /**
         * Cannot use `styled` to override this because the specificity of EuiHorizontalRule's
         * CSS selectors is too high.
         */
        marginLeft: `-${euiVars.euiPanelPaddingModifiers.paddingMedium}`,
        marginRight: `-${euiVars.euiPanelPaddingModifiers.paddingMedium}`,
        /**
         * The default width is 100%, but this should be greater.
         */
        width: 'auto',
      }}
```